.github/workflows/pr.yml

```diff
@@ -590,3 +484,66 @@ jobs:
         if: ${{ success() }}
         run: echo ${{ steps.publish-generator-coverage.outputs.coveralls-api-result }}
         working-directory: ./generator/
+
+# release
+  publish-artifacts:
+    runs-on: ubuntu-latest
+    name: Publish Release artifacts
+    if: ${{ success() && github.event_name == 'push' && github.ref_type == 'tag' }}
+    needs:
+      - tests-linux
+      - flutter-linux
+      - tests-windows
+      - flutter-windows
+      - tests-macos
+      - flutter-macos
+      - flutter-android
+      - flutter-ios
+      - generator
+    steps:
```

Comment:
It can, it's just going to be super inefficient - downloading an artifact will likely take a couple of seconds - this is not building anything, just downloading what was already there.